### PR TITLE
Bandwidth / Data usage reduction

### PR DESCRIPTION
**Pull request by lifeeth**
Adds Bandwidth / Data usage reduction docs

### CODE REVIEW

File: pages/runtime/bandwidth-reduction.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Controlling device's Bandwidth / Data usage.
+
+In order to provide our users with a clear view of their devices, the resin device Agent constantly keeps our API informed about the devices' conditions and applies any changes, like downloading new application updates or environment variable changes by negotiating with our API. While a quick reflection of actions is great during development, it comes at a cost of increased data usage. 
+
+In order to give our power users control over data flow we enabled a few RESIN [environment variables](http://docs.resin.io/#/pages/management/env-vars.md).
```
Comment (pcarranzav) [inline pages/runtime/bandwidth-reduction.md]: Might be better to specify it's Config Vars (dunno if the UX for this is changing, but the difference is worth noting)

File: pages/runtime/bandwidth-reduction.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+|----------------------------|------------------|-----------------------------------------------------
+| `RESIN_VPN_CONTROL`        | true/false       |  Enable / Disable VPN
+| `RESIN_CONNECTIVITY_CHECK` | true/false       |  Enable / Disable connectivity check VPN is disabled
+| `RESIN_POLL_INTERVAL`      | 6000 to 86400000 |  Resin API Poll interval
+| `RESIN_LOG_CONTROL`        | true/false       |  Enable / Disable logs from being sent to Resin
+
+Side-effect/Warning
+-------------------
+
+`RESIN_VPN_CONTROL`: This defines the ability to send instantaneous updates to the device. Turning off the VPN means that any application or environment variable update is reflected only when the device polls for these changes. The Web Terminal does not function when the VPN is disabled.
```
Comment (pcarranzav) [inline pages/runtime/bandwidth-reduction.md]: This would also disable Public URL, wouldn't it?

Comment (pcarranzav) [inline pages/runtime/bandwidth-reduction.md]: And our ability to recover a device if the supervisor crashes...

File: pages/runtime/bandwidth-reduction.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Controlling device's Bandwidth / Data usage.
+
+In order to provide our users with a clear view of their devices, the resin device Agent constantly keeps our API informed about the devices' conditions and applies any changes, like downloading new application updates or environment variable changes by negotiating with our API. While a quick reflection of actions is great during development, it comes at a cost of increased data usage. 
+
+In order to give our power users control over data flow we enabled a few RESIN [environment variables](http://docs.resin.io/#/pages/management/env-vars.md).
+
+|Variable                    | Allowed Value    |   Action
```
Comment (pcarranzav) [inline pages/runtime/bandwidth-reduction.md]: Would be good to also specify default values

**Comment by pcarranzav** [comment]
Nice one @lifeeth - can you also doc this in the [supervisor repo](https://github.com/resin-io/resin-supervisor/tree/master/docs)? I'd like to document all the config variables, and this looks like a great start.

File: pages/runtime/bandwidth-reduction.md (hunk 2 above)
Comment (lifeeth) [inline pages/runtime/bandwidth-reduction.md]: Yes it would

File: pages/runtime/bandwidth-reduction.md (hunk 3 above)
Comment (lifeeth) [inline pages/runtime/bandwidth-reduction.md]: Added.

**Comment by lifeeth** [comment]
@pcarranzav We will need to split the docs into supervisor so that we dont have docs in two places - I think this will need more rework of the entire docs structure.

**Comment by pcarranzav** [comment]
@lifeeth indeed. This relates to #168 (cc @shaunmulligan)

On another note, env var names here should also be changed to use the `RESIN_SUPERVISOR_` namespace ;)

**Comment by pcarranzav** [comment]
Oh, I see you already changed it, nevermind :)

**Comment by lifeeth** [comment]
Can we merge this? and do the restructure?

**Pull request merged**